README.md

```diff
@@ -1,35 +1,54 @@
 # Node-RED-Docker
 
-[![Greenkeeper badge](https://badges.greenkeeper.io/node-red/node-red-docker.svg)](https://greenkeeper.io/)
+[![Greenkeeper badge](https://badges.greenkeeper.io/RaymondMouthaan/node-red-docker.svg)](https://greenkeeper.io/)
+[![Build Status](https://travis-ci.org/RaymondMouthaan/node-red-docker.svg?branch=master)](https://travis-ci.org/RaymondMouthaan/node-red-docker)
+[![DockerHub Pull](https://img.shields.io/docker/pulls/raymondmm/node-red.svg)](https://hub.docker.com/r/raymondmm/node-red/)
 
-This project describes some of the many ways Node-RED can be run under Docker.
-Some basic familiarity with Docker and the
-[Docker Command Line](https://docs.docker.com/engine/reference/commandline/cli/)
-is assumed.
+This project describes some of the many ways Node-RED can be run under Docker and has support for multiple architectures (amd64, arm32v6, arm23v7 and arm64v8).
+Some basic familiarity with Docker and the [Docker Command Line](https://docs.docker.com/engine/reference/commandline/cli/) is assumed.
 
-This project also provides the build for the `nodered/node-red-docker`
-container on [DockerHub](https://hub.docker.com/r/nodered/node-red-docker/).
+This project also provides the build for the `nodered/node-red`
+container on [DockerHub](https://hub.docker.com/r/raymondmm/node-red/).
 
-To run this directly in docker at it's simplest just run
+## Quick Start
+To run this directly in docker at it's simplest just run:
 
-        docker run -it -p 1880:1880 --name mynodered nodered/node-red-docker
+        docker run -it -p 1880:1880 --name mynodered nodered/node-red
 
-Let's dissect that command...
+Let's dissect that command:
 
-        docker run      - run this container... and build locally if necessary first.
-        -it             - attach a terminal session so we can see what is going on
-        -p 1880:1880    - connect local port 1880 to the exposed internal port 1880
-        --name mynodered - give this machine a friendly local name
+        docker run              - run this container... and build locally if necessary first.
+        -it                     - attach a terminal session so we can see what is going on
+        -p 1880:1880            - connect local port 1880 to the exposed internal port 1880
+        --name mynodered        - give this machine a friendly local name
         nodered/node-red-docker - the image to base it on - currently Node-RED v0.14.5
 
 
-Running that command should give a terminal window with a running instance of Node-RED
+Running that command should give a terminal window with a running instance of Node-RED.
 
         Welcome to Node-RED
+
         ===================
-        8 Apr 12:13:44 - [info] Node-RED version: v0.14.5
-        8 Apr 12:13:44 - [info] Node.js  version: v4.4.7
-        .... etc
+
+        6 May 00:36:30 - [info] Node-RED version: v0.18.4
```

Comment:
Should at least match latest versions for now :-)

Reply:
Taking care of this in the new coming PR.